stable/database/values.yaml

```diff
@@ -393,6 +397,10 @@ database:
           accessModes:
             - ReadWriteOnce
           # storageClass: "-"
+
+          # The data source for the archive PVC
```

Comment:
This is the journal PVC for hot copy SMs.

Reply:
Oops. A good reason to not make one more change after a log day of debugging.